FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -93,6 +112,90 @@ const UniversalFeed = () => {
     return getFeedResponse;
   }
 
+  // this useEffect handles the execution of addPost api
+  useEffect(() => {
+    if (
+      mediaAttachmemnts.length > 0 ||
+      linkAttachments.length > 0 ||
+      postContent != ''
+    ) {
+      setPostUploading(true);
+      postAdd();
+    }
+  }, [mediaAttachmemnts, linkAttachments, postContent]);
+
+  // aws configuration
+  function getAWS() {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    (AWS.config.region = process.env.AWS_REGION),
+      (AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+        IdentityPoolId: `${process.env.AWS_IDENTITY_POOL_ID}`,
+      }));
+    const s3 = new AWS.S3({
+      apiVersion: process.env.S3_API_VERSION,
+      params: {Bucket: process.env.S3_BUCKET},
+    });
+    return s3;
+  }
```

Comment:
move this aws configuartion to a util file.

Reply:
resolved